dash_docs/chapters/external_resources/examples/addingExternalCSS.R

```diff
@@ -2,14 +2,14 @@ library(dash)
 library(dashHtmlComponents)
 
 external_scripts <- list(
-  list('https://www.google-analytics.com/analytics.js'),
+  list(src = 'https://www.google-analytics.com/analytics.js'),
```

Comment:
Is it necessary in dash for R to wrap these in `list(src = ...)`? On the Python side, if all you have is a `src` (or an `href` for `external_stylesheets`), you can just pass in a string for each script. The only reason you need the list is if you have other attributes (`integrity`, `crossorigin` etc).

Reply:
In Dash for R it is necessary for them to be named elements. The relevant bit of code I found was:

https://github.com/plotly/dashR/blob/add8dd3d93b23c0feb966e39f711ed58be256373/R/utils.R#L679

Reply:
A little farther down though it looks like a plain string is accepted
https://github.com/plotly/dashR/blob/add8dd3d93b23c0feb966e39f711ed58be256373/R/utils.R#L690
and in constructing the html it seems like there's always an `if(is.list(...))`
https://github.com/plotly/dashR/blob/add8dd3d93b23c0feb966e39f711ed58be256373/R/utils.R#L621-L623
Not a big deal, mostly I just want to know whether this behavior matches Python, but if we do have the simpler version it would be nice to use it when it fits.

Reply:
To be clear: I'm not saying the original form is correct, I'm suggesting it looks like a bare string would work
```suggestion
  'https://www.google-analytics.com/analytics.js',
```

Reply:
And as a separate note: did you make this change because the unnamed list threw an error, or because the desired script just didn't show up (or didn't load correctly) on the page? If the latter, perhaps `assertValidExternals` should be improved to catch this case.

Reply:
In this case, I made the change because of the `"A valid URL must be included with every entry in external_scripts. Please sure no 'src' entries are missing or malformed."` error. And I ran into a similar one for the `href` elements. I'll take another look and see what happens in `assertValidExternals`, because it does seem like it _should_ work.

Reply:
Bare strings do work :+1: 

Added/fixed in 3454976.